src/logic/operate.js

```diff
@@ -0,0 +1,17 @@
+import Big from 'big';
+
+export default function operate({ numberOne, numberTwo, operation }) {
+  let result;
+  if (operation === '÷') {
+    result = Big(numberOne) / Big(numberTwo);
```

Comment:
Big.js has a plus function, likewise a div, minus, times and others please use that to perform operations
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)

Reply:
Nice, I didn't know about these functions inside big.js!